deploy/service-assurance/prometheus/prometheus.yaml

```diff
@@ -18,15 +16,15 @@ spec:
       port: metrics
   podMetadata:
     labels:
-      sa-app: prometheus-white
+      app: prometheus
+      sa-affinity: white
```

Comment:
Our discussion proposes that we drop affinity

Reply:
fixed